bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -19,6 +19,9 @@
 
 {% block extrahead %}
   {% stylesheet 'partners' %}
+<!--[if IE 9]>
+  {% javascript 'matchmedia_addlistener' %}
```

Comment:
This is being inserted into the `<head>`, so it's coming before the [regular polyfill](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/base-resp.html#L151). 

IE9 throws a sad `SCRIPT438: Object doesn't support property or method 'matchMedia' `